website/docs/r/app_configuration.html.markdown

```diff
@@ -0,0 +1,79 @@
+---
+subcategory: "App Configuration"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_configuration"
+sidebar_current: "docs-azurerm-resource-app-configuration"
+description: |-
+  Manages an Azure App Configuration.
+
+---
+
+# azurerm_container_registry
```

Comment:
Copy/paste error here? This should be `#azurerm_app_configuration`

Reply:
Correct, fixed with #5186.